components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -254,6 +294,160 @@
     newCustomField = { ...emptyCustomField };
     newFieldTitleElement.focus();
   }
+
+  type CustomFieldWithId = CustomField & { id: string };
```

Comment:
Do we still need this type? Looks like you ended up adding `id` to `CustomField`'s props.

Reply:
Good catch, removing id from `CustomField`!